Sources/TokamakDOM/Shapes/Path.swift

```diff
@@ -50,7 +50,7 @@ extension Path: ViewDeferredToRenderer {
         "rx": "\(roundedRect.cornerSize.width)",
         "ry": "\(roundedRect.style == .continuous ? roundedRect.cornerSize.width : roundedRect.cornerSize.height)",
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakDOM/Shapes/Path.swift;
  Line: 51;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 115 characters (`line_length`)